src/index.js

```diff
@@ -1,12 +1,61 @@
-import _ from 'lodash';
-import './style.css'
+/* eslint-disable no-plusplus */
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] The last one please remove these disablers as well.


You also can these disablers as well. By running:
```
npx eslint . --fix
```

in your terminal. This command fix both above eslint.. issue.